docs/src/overview.md

```diff
@@ -0,0 +1,65 @@
+# Overview of the structure of Trixi
+
+Trixi is designed as a library of components for discretizations of hyperbolic
+conservation laws. Thus, it is not a classical monolithic PDE solver often found
+in legacy codes that might be configured at runtime via parameter files.
+Instead, each simulation is configured by pure Julia code. Many examples of such
+simulation setups, called *elixirs* in Trixi, are provided in the
+[examples](https://github.com/trixi-framework/Trixi.jl/blob/main/examples)
+folder.
+
+Trixi uses the method of lines. Thus, the main ingredients of an elixir designed
+to solve a PDE numerically are the spatial semidiscretization and the time
+integration scheme.
+
+
+## Semidiscretizations
+
+Semidiscretizations are high-level descriptions of spatial discretizations
+specialized to certain PDEs. Trixi's main focus is on hyperbolic conservation
+laws represented in a [`SemidiscretizationHyperbolic`](@ref).
+Such semidiscretizations are usually called `semi` in Trixi.
+
+The basic building blocks of a semidiscretization are
+
+- a `mesh` describing the geometry of the domain
+- a set of `equations` describing the physical model
+- a `solver` describing the numerical approach
```

Comment:
Hmm, it's still only a model and not the "complete" physics of the problem, I think.

Reply:
Hm. I don't really like the term `physical model`, since IMHO it is a *mathematical* model of a physical system. But that's bikeshedding and can be closed...